manifests/init.pp

```diff
@@ -12,6 +12,8 @@
 #
 # $enable_deb::                                Enable debian content plugin
 #
+# $enable_ansible::                            Enable ansible content plugin
```

Comment:
Is the discrepancy in naming intentional? In the Katello application itself, the setting is `enable_ansible_collection` (https://github.com/theforeman/puppet-katello/pull/401/files#diff-510884a7414666a0d988133c5f869510210d883aa1580d1248e74f1d16ca7d91R13)

Reply:
yes, this is intentional.  the pulp plugin is called 'ansible', but provides two different types of 'repositories' (really remotes/distributors), ansible_collections and ansible_roles.  Katello only supports ansible_collections at this time, and would likely implement roles as a different repo type (if we ever implement support)